packages/core/src/App/Components/Layout/Header/dtrader-v2/account-switcher-dtrader-v2.tsx

```diff
@@ -213,7 +214,13 @@ const AccountSwitcherDTraderV2 = observer(({ history }: TAccountSwitcherDTraderV
         <div className='acc-switcher-dtrader__wrapper'>
             {is_landing_company_loaded ? (
                 <React.Fragment>
-                    <div className='acc-switcher-dtrader__accounts-list'>
+                    <div
+                        className={classNames('acc-switcher-dtrader__accounts-list', {
+                            'acc-switcher-dtrader__accounts-list--with-button': show_button,
```

Comment:
This is only part of improvement for Action sheet, the rest will be done in a separate PR to Quill UI library. Not related to current task, just a convenient opportunity to deliver at least a part of the fix